patches/master_patch.patch

```diff
@@ -1842,6 +1842,45 @@ index b73f65168086189aed021cc82ffef5fde1f2d061..aa3f97fa57a3cb5616cd3811a34d0698
    return SimplifyUrlForRequest(url).spec();
  }
  
+diff --git a/net/http/transport_security_state_static.json b/net/http/transport_security_state_static.json
+index 698980a564bc1e72bdc3967244fcba0dcd610af5..549b3a6de5313b729727385a47de2675e7f9bfb3 100644
+--- a/net/http/transport_security_state_static.json
++++ b/net/http/transport_security_state_static.json
+@@ -55,6 +55,19 @@
+         "TestSPKI"
+       ]
+     },
++    {
++      "name": "brave",
++      "static_spki_hashes": [
++        "DigiCertGlobalRoot",
++        "DigiCertEVRoot",
++        "DigiCertAssuredIDRoot",
++        "GlobalSignRootCA",
++        "GlobalSignRootCA_R2",
++        "GlobalSignRootCA_R3",
++        "LetsEncryptAuthorityPrimary_X1_X3",
++        "LetsEncryptAuthorityBackup_X2_X4"
++      ]
++    },
+     {
+       "name": "google",
+       "static_spki_hashes": [
+@@ -252,6 +265,14 @@
+     { "name": "preloaded-expect-staple.badssl.com", "expect_staple": true, "expect_staple_report_uri": "https://report.badssl.com/expect-staple" },
+     { "name": "preloaded-expect-staple-include-subdomains.badssl.com", "expect_staple": true, "expect_staple_report_uri": "https://report.badssl.com/expect-staple", "include_subdomains_for_expect_staple": true },
+ 
++    //Brave
++    { "name": "ledger.mercury.basicattentiontoken.org", "mode": "force-https", "pins": "brave"},
```

Comment:
the ledger client also reaches out to `balance.mercury.basicattentiontoken.org`

Reply:
there are also `-staging` equivalents

Reply:
Will update, what about `publishers.basicattentiontoken.org`?

Reply:
i don't believe that domain is used in browser-laptop

Reply:
I think we need to find a better way to handle this because this patch is way too big. If we're going to make this many changes we would be better off patching `net/http/transport_security_state.h` to use our `kDefaultHSTSSource`